central/notifiers/jira/jira.go

```diff
@@ -15,13 +15,14 @@ import (
 
 	jiraLib "github.com/andygrunwald/go-jira"
 	"github.com/pkg/errors"
-	mitreDataStore "github.com/stackrox/rox/central/mitre/datastore"
 	namespaceDataStore "github.com/stackrox/rox/central/namespace/datastore"
 	"github.com/stackrox/rox/central/notifiers"
 	"github.com/stackrox/rox/generated/storage"
 	"github.com/stackrox/rox/pkg/errorhelpers"
 	"github.com/stackrox/rox/pkg/httputil/proxy"
 	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/mitre/datastore"
+	mitreDataStore "github.com/stackrox/rox/pkg/mitre/datastore"
```

Comment:
Looks like we should only need 1 of these?

Reply:
Good catch, fixed